paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/FlinkConnectorOptions.java

```diff
@@ -301,6 +301,14 @@ public class FlinkConnectorOptions {
                     .withDescription(
                             "Sink committer memory to control heap memory of global committer.");
 
+    public static final ConfigOption<Integer> LIMIT_PARTITION_REQUEST =
```

Comment:
Can this be a generic option for all compute engines?

Reply:
ok。I will add it to `CoreOptions`, and support spark.

Reply:
I updated it